http/server.go

```diff
@@ -33,19 +34,27 @@ func (s *Server) Start() error {
 
 func (s *Server) init() {
 	s.router = mux.NewRouter().StrictSlash(true)
-	s.registerRoutes()
+
 	s.router.Use(httplog.Request)
+
+	s.registerRoutes()
+
 	s.Handler = s.router
 }
 
+const alphanum20 = "[a-zA-Z0-9]{2,20}"
+
 func (s *Server) registerRoutes() {
 	s.router.HandleFunc("/", handleRoot)
-	s.router.HandleFunc("/report", s.handleGetReport).Methods("GET")
-	s.router.HandleFunc("/report", s.handlePostReport).Methods("POST")
+
+	s.router.HandleFunc("/benchmark", s.handleRetrieveBenchmark).Methods("POST")
+
+	s.router.HandleFunc("/benchmark", s.handleCreateBenchmark).Methods("GET").
```

Comment:
I realised that this change breaks the runner.
I'll keep it here because it makes more sense, but this opens the question of versioning our server endpoints and whether or not the runner must use env variables to provide the endpoint to call.

Reply:
- Versioning the endpoints is a perfectly suited solution for this, because we need the older endpoints to be always available for the runner
- We'll never use environment variables for the runner's target endpoint because they're user-defined and asking them to provide our new endpoint because we casually deleted the previous one doesn't sound like a great idea 😕